main.py

```diff
@@ -1,10 +1,69 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+
+    def create_guild(guild: dict) -> None:
+        if guild and not Guild.objects.filter(name=guild["name"]).exists():
+            Guild.objects.create(
+                name=guild["name"],
+                description=guild["description"]
+            )
```

Comment:
try to use get_or_create method to simplify ur code
the same is for similar cases